### DISCUSSION TITLE
desktop widget

### DISCUSSION THREAD

**Discussion by Boomz123** [breezy-weather/breezy-weather#939] [Q&A] [answered]
Desktop widget, clock+weather (vertical display), will flash back, and the settings page cannot run either

**Comment by Boomz123** [comment, marked as answer]
> Desktop widget, clock+weather (vertical display), will flash back, and the settings page cannot run either桌面小部件，时钟+天气（垂直显示），将闪回，设置页面也无法运行

5.1.8

**Comment by papjul** [comment]
?

**Comment by Boomz123** [comment]
It is actually raining, but the card indicates that there will be no rain for the next 120 minutes.
![Screenshot_20240420_193217](https://github.com/breezy-weather/breezy-weather/assets/166996243/037bc4d0-4b13-4736-a3eb-9eb272cd9dd8)

**Reply by papjul** [reply]
One is current, the other is the MinuteCast. AccuWeather is responsible for the discrepancy between the two.
